winui/Kanban/cards.md

```diff
@@ -0,0 +1,516 @@
+---
+layout: post
+title: Cards in WinUI Kanban control | Syncfusion
+description: This section describes about cards in Syncfusion WinUI Kanban (SfKanban) control, its elements and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Cards in WinUI Kanban (SfKanban) control
+
+The Kanban cards visually represent tasks and their progression through various stages. The default UI of each card can be customized using the following properties of the KanbanModel.
+
+* `Title` - Sets the title of the card.
+* `Description` - Sets the description text of the card.
+* `Category` - Sets the category of the card, determining which column it will be placed in.
+* `Assignee` - Defines the assignee associated with the card in the `KanbanModel`.
+* `Image` - Sets the image for the card, which is displayed on the right side in the default card template.
+* `IndicatorColorKey` - Specifies the indicator color for the card, commonly used for categorize or prioritize card based on color key values.
+* `Id` - Sets the unique ID of the card.
+* `Tags` - Specifies the tags for the card, displayed at the bottom of the default card template.
+
+Below is an example that shows how to define values in the `KanbanModel`.
+
+{% highlight C# %}
+
+KanbanModel taskDetail = new KanbanModel()
```

Comment:
@TamilarasanGunasekaran , add example as i shared in prev response

Reply:
Updated.